src/dataset_info.rs

```diff
@@ -38,23 +38,22 @@ pub fn create_dataset_info(output_path: Option<PathBuf>) -> std::io::Result<()>
     };
 
     // Get the file path for the dataset_info toml file
-    let path = output_path.unwrap_or_else(|| PathBuf::from("dataset_info.toml"));
+    let path = output_path.unwrap_or_else(|| PathBuf::from("."));
     let file_path = path.join("dataset_info.toml");
 
     // Check if the file already exists
     if file_path.exists() {
         eprintln!("File '{}' already exists. Aborting.", file_path.display());
-        return Err(std::io::Error::new(
-            std::io::ErrorKind::AlreadyExists,
-            "File already exists",
-        ));
+
+        // return ok if the file exist.
+        return Ok(false);
```

Comment:
Instead of returning a bool, returning the "AlreadyExists" error is okay -- just add a specific case to the matching in src/lib.rs to specifically let the "AlreadyExists" error pass without printing anything.